### PR TITLE
Fix settings page sidebar

### PR DESCRIPTION
**Pull request by ayumihamsaki**
What have I done here?

Well I basically went hardcore and deleted a ton of complete crap out of the css and less files and added the correct code.

The issue, there was a bunch of conflicting media queries, classes that created a huge gap on the right hand side of the backend. You would either see it in a flash when loading or see the top navigation menu load on the left hand side of the screen and then get over-written to cover the rest of the screen. Someone tried to merge the sidebar and the top navigation code together.

Basically it was a total mess and totally unless code. So I deleted it. Then added a few lines of clean code and the result is this:

ezgif-2-087b529c55fc

Fixes this github issue: https://github.com/octobercms/october/issues/4505

Testing, normally doing something like this would break tihngs, but I'm so sure that the code I delete was total garbage, I'm sure the testers will not find errors or anything breaking - that's how bad the code was.

### CODE REVIEW

**Comment by Samuell1** [comment]
It totally changes how it looks and worked before

**Comment by ayumihamsaki** [comment]
I forgot to mention, if people want a bigger sized sidebar in the settings page, then adding a drag bar to extend this pr is my intention, see here: https://github.com/octobercms/october/issues/3971

Also on top of the dragbar, I would want to remove the display table inline rows and switch to using flexbox wrap to give a smooth feeling when a user adjusts the sidebar using a dragbar.

**Comment by Samuell1** [comment]
But still it should be same size with or without dragbar as it was before, you still have empty right side of page why not to use that and in your gif there is logo getting smaller but doesnt hide on mobile resolutions.

**Comment by ayumihamsaki** [comment]
As I said there are many issues with the current setup and it's all over the place! You suggest it should be the same size as before, but in truth it's not the same size because when you first open the settings page you get three columns then select something and get a single column. I don't know any professional website that has moving menu's all over the place, when a user selects something!

Here I am creating a more stable thing, where a user opens the settings page for the first time and gets one size and then opens the next thing in that section and gets the same sized sidebar.

Adding a dragbar to this pr the user can get a large sidebar when opening the settings page and clicking the next thing and get the same large sidebar.

Lastly, sorry but no! I am not going to add a dragbar to a moving all over the place sidebar. Having two things moving against each other would create an even worse mess!

Having a stable sidebar and adding a dragbar to a stable object allows a stable outcome!

It's a shame your gif is moving so fast and tries to cover up all the bugs, that I summarized above.

**Comment by Samuell1** [comment]
Yes its bad but dont forget this is administration sidebar not a website menu, if there will be many items it will be a lot longer and it will take a lot of time to get where you want.

I am mostly showing with that gif that there are more items on bigger screen showed at once to faster navigate at first open and showing that on mobile there isnt any extremly tiny logo on right side.

**Comment by ayumihamsaki** [comment]
I'm my idea is baised on two things, the first is your wonderful suggestion of adding a session storage to a dragbar and the second is a guess that the reason the rest of the settings sections have a sidebar set at 300px is not a width, but more of a min-width. I'm guessing it's setup that way to work on all web pages in the settings page using the min width of the sidebar.

So my idea is basically to have a screensize above 400px uses a min-width of 300px and then a dragbar can be used to increase the sidebar width - according to the user's spec. and then finally your idea of a session storage to auto save the sidebar width for future setting page loads. Then lastly have flexbox wrap to display the sidebar columns smoothly e.g. 1 column, 2 columns, 3 columns or even more for uses with 4k and upwards computer screens etc.

I understand what you are saying and it sounds like you want a 3 column layout for your personal websites. I don't mind putting this pr on hold until the dragbar pr is completed and then merging the two at the same time - thus giving you the increased sidebar width.

**Comment by ayumihamsaki** [comment]
My idea is baised on two things, the first is your wonderful suggestion of adding a session storage to the dragbar and the second is a guess that the reason the rest of the settings sections have a sidebar set at 300px is not a width, but more of a min-width. I'm guessing it's setup that way to work on all web pages in the settings page using the min width of the sidebar.

So my idea is basically to have a screensize above 400px using a sidebar with a min-width of 300px and then a dragbar which can be used to increase the sidebar width - according to the user's spec. Then your idea of a session storage to auto save the sidebar width for future setting page loads. Then lastly have flexbox wrap to display the sidebar columns smoothly e.g. 1 column, 2 columns, 3 columns when the sidebar is increased manually in size as per the user spec.

This way it hopefully gives the user more control and a better stable sidebar.

I understand what you are saying and it sounds like you want a 3 column layout for your personal websites. I don't mind putting this pr on hold until the dragbar pr is completed and then merging the two at the same time - thus giving you the increased sidebar width.

**Comment by ayumihamsaki** [comment]
@Samuell1 initial coding of the second part of this pr idea:

ezgif-2-3c1e12571716

**Comment by daftspunk** [comment]
Most of the faults here are actually part of the design. Ghost/Ayumi: Please message me on Slack (spunky) and I can explain why this is not accepted.
